src/reanimated2/core.ts

```diff
@@ -371,7 +371,7 @@ if (!NativeReanimatedModule.useOnlyV1) {
       : (workletValueSetterJS as <T>(value: T) => void)
   );
 
-  if (!isWeb() && isConfigured()) {
+  if (!isWeb() && !isChromeDebugger() && isConfigured()) {
```

Comment:
i don't know much about the console mocking here, we could also just mock global.performance if really needed:

```js
if (!global.performance && global._chronoNow) {
        global.performance = {
            now: global._chronoNow,
        }
};
```

Reply:
even without chrome debugger `global.performance` and `global._chronoNow` seem both to be undefined. should we remove the mocke in general? I can't see any purpose for it